etomica-apps/src/main/java/etomica/osmoticvirial/MCMoveGeometricClusterRestrictedGE.java

```diff
@@ -71,7 +71,7 @@ public MCMoveGeometricClusterRestrictedGE(PotentialMasterCell potentialMaster, S
         clusterAtomsList1 = new ArrayList<>();
         clusterAtomsList2 = new ArrayList<>();
         jNeighbors = new ArrayList<>();
-        neighbors = new Api1ACell(space.D(), potentialMaster.getRange(), potentialMaster.getCellAgentManager());
+        neighbors = new Api1ACell(space.D(),neighborRange,potentialMaster.getCellAgentManager());
```

Comment:
This doesn't compile now. Is there supposed to be a variable `neighborRange`?

Reply:
Sorry. the previous one that had potentialMaster.getRange() was correct.